modules/django/Makefile

```diff
@@ -1,7 +1,12 @@
 ## Run Django tests
 django/test: pipenv
 	$(WITH_PIPENV) ./manage.py migrate
-	$(WITH_PIPENV) ./manage.py test
+	$(WITH_PIPENV) ./manage.py test -- -m "not postbuild"
+
+## Run Django tests
+django/test-post-build: pipenv
+	$(WITH_PIPENV) ./manage.py migrate
+	$(WITH_PIPENV) ./manage.py test -- -m "postbuild"
```

Comment:
This should be a separate merge request. However, I'd advise that instead of making this change, update your repo to use pytest directly instead of django's management command.